apps/src/code-studio/pd/application_dashboard/application_dashboard.jsx

```diff
@@ -28,51 +28,60 @@ const ApplicationDashboardHeader = (props) => (
 );
 
 const paths = {
-  'csf_facilitators': 'CSF Facilitators',
+  'csf_facilitators': 'CSF Facilitator Applications',
   'csd_facilitators': 'CSD Facilitator Applications',
   'csp_facilitators': 'CSP Facilitator Applications',
   'csd_teachers': 'CSD Teacher Applications',
   'csp_teachers': 'CSP Teacher Applications'
 };
 
-const ApplicationDashboard = ()=> (
-  <Router history={browserHistory} >
-    <Route path="/" component={ApplicationDashboardHeader}>
-      <IndexRedirect to="/summary"/>
-      <Route
-        path="summary"
-        breadcrumbs="Summary"
-        component={Summary}
-      />
-      {
-        _.flatten(Object.keys(paths).map((path, i) => {
-          return [
-            (
-              <Route
-                key={`detail_${i}`}
-                path={`${path}/(:applicationId)`}
-                breadcrumbs={[
-                  {name: paths[path], path: path},
-                  {name: 'Application Details', path: ''}
-                ]}
-                component={DetailView}
-                title={'Application Detail'}
-              />
-            ),
-            (
-              <Route
-                key={`quick_view_${i}`}
-                path={path}
-                breadcrumbs={paths[path]}
-                component={QuickView}
-                title={paths[path]}
-              />
-            )
-          ];
-        }))
-      }
-    </Route>
-  </Router>
-);
+export default class ApplicationDashboard extends React.Component {
+  static propTypes = {
+    partnerName: PropTypes.string
+  };
+
+  render() {
+    const allPartners = "All Regional Partner Applications";
```

Comment:
You don't need to define this as a const - AFAIK you are only using this here

Reply:
Actually - I would prefer regional partner name as an optional prop - in quick_view, use || to handle it being unset

Reply:
quick view and summary view both get passed `regionalPartner={this.props.partnerName || allPartners}` (lines 53 and 75). Should I make it an optional prop in both components, or create a little method on application dashboard to determine the value for both?